example-playbooks/replace_node/replace_node.yml

```diff
@@ -155,7 +155,7 @@
 
     - name: Wait for the added node to become healthy
       shell: |
-        nodetool status|grep -E '^UN'|grep "{{ vars['ansible_'~scylla_nic].ipv4.address }}"| wc -l
+        nodetool status|grep -E '^UN'|grep " {{ vars['ansible_'~scylla_nic].ipv4.address }} "| wc -l
```

Comment:
Why not using `grep -w ` instead?

Reply:
I wasn't familiar with `grep -w`. It's indeed a better option.
Thanks.